angularjs-portal-home/src/main/webapp/samples/notifications.json

```diff
@@ -21,6 +21,14 @@
         "title"  : "You need to modify your NetID account to activate essential UW Services.",
         "actionURL" : "https://www.mynetid.wisc.edu/modify",
         "actionAlt" : "Activate Services"
+      },
+      {
+        "id"     : 4,
+        "groups" : ["Facstaff","Portal Administrators"],
+        "title"  : "Employee benefits open enrollment for 2016 starts Oct 5th, 2015 and ends Oct 30th, 2015.",
```

Comment:
From this morning's f2f over at Service Center:

* Prefer "Annual Benefits Enrollment" to "open enrollment".
* Style guidance: prefer "October 5" to "Oct 5th"

So,

`"Employee Annual Benefits Enrollment for 2016 starts October 5, 2015 and ends October 30, 2015."`

and likewise

"View news article on Annual Benefits Enrollment".